deployment/huggingface/yolos-seq2seq.ipynb

```diff
@@ -0,0 +1,1705 @@
+        "outputId": "047eb733-5239-4de7-e43e-e374a81abdff"
+      },
+      "outputs": [],
+      "source": [
+        "yolos = Yolos()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "R6jR843MiXtc"
+      },
+      "outputs": [],
+      "source": [
+        "url = yolos.example()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "6lMHliQ1iY2Y"
+      },
+      "outputs": [],
+      "source": [
+        "predictions = yolos.predict(url)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Z4gX4jxvkFA8"
+      },
+      "outputs": [],
+      "source": [
+        "results_dfs = []\n",
+        "\n",
+        "for result in predictions[0]:\n",
+        "    cols = list(result.keys())[:-1]\n",
+        "    box = list(result['box'].keys())\n",
+        "    cols.extend(box)\n",
+        "    data = []\n",
+        "\n",
+        "    for k, v in result.items():\n",
+        "        data.extend(list(v.values())) if k == 'box' else data.append(v)\n",
+        "    \n",
+        "    results_dfs.append(pd.DataFrame([data], columns = cols))"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Nb7dcFUsiZC4"
+      },
+      "outputs": [],
+      "source": [
+        "df = pd.concat(results_dfs)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 426
+        },
+        "id": "IVCKPhFMq25v",
+        "outputId": "e0cdf59d-1828-42b7-807c-6c63be974a57"
+      },
+      "outputs": [],
+      "source": [
+        "df"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "KgGilFLFfgiw"
+      },
+      "source": [
+        "## 6. Model Register"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "gqTwdp7lhVlS",
+        "outputId": "b8cc3f44-0356-4020-ce94-585add15e8ef"
+      },
+      "outputs": [],
+      "source": [
+        "registered_model = client.get_or_create_registered_model(name='YOLOS - Hugging Face')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "iDCNSz-zhjdj",
+        "outputId": "66babeb0-a96b-4a24-e245-690417a048e0"
+      },
+      "outputs": [],
+      "source": [
+        "model = registered_model.create_standard_model(\n",
+        "      name = 'v0',\n",
+        "      model_cls = Yolos,\n",
+        "      model_api = ModelAPI([{'urls': url}], df),\n",
```

Comment:
As per [my comment on another PR](https://github.com/VertaAI/examples/pull/50#discussion_r1056713622), `ModelAPI` must match the exact input and output of the model. Although for this notebook, I think we could actually use

```python
ModelAPI(yolos.example(), yolos.predict(yolos.example()))
```

Reply:
When using `ModelAPI(yolos.example(), yolos.predict(yolos.example()))`, the Model input and output labels are not shown in the WebApp.

A little tweak made it work: `ModelAPI([{'urls': yolos.example()}], yolos.predict(yolos.example())[0])`